ateam_ui/CMakeLists.txt

```diff
@@ -0,0 +1,59 @@
+cmake_minimum_required(VERSION 3.5)
+project(ateam_ui)
+
+find_package(ament_cmake REQUIRED)
+find_package(rosbridge_server REQUIRED)
+
+install(DIRECTORY
+  launch
+  DESTINATION share/${PROJECT_NAME}
+  USE_SOURCE_PERMISSIONS
+)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
+  add_subdirectory(test)
+endif()
+
+
+# Get recursive list of files in ./src
+file(GLOB_RECURSE src_files
+  LIST_RECURSE true
+  RELATIVE ${CMAKE_CURRENT_SOURCE_DIR}/src
+  ${CMAKE_CURRENT_SOURCE_DIR}/src/*
+)
```

Comment:
That would be an issue. Is there a way to have it pick up new files? The reason that I'm doing all that in the first place is because the /src file has to be copied to the install location before building it.

Reply:
I believe you can just copy the folder instead of a list of files. The source arg just has to be a directory. See page 222 ish here https://drive.google.com/file/d/1z48ybhezGLe4_Xox-euVkHv68hWMjhBz/view?usp=sharing

Reply:
tldr depending on how you want to deal with the naming
```
file(COPY base/srcDir DESTINATION destDir) # --> destDir/srcDir
file(COPY base/srcDir/ DESTINATION destDir) # --> destDir
```

Reply:
I don't think that works for detecting new files or even changes to existing ones, thats's why I was using the configure_file command, that one actually detects changes to the files

Reply:
Ah shoot, you right. It'll probably end up being some really gross custom target thing then. It's fine for now and if it becomes a problem we can always fix it